indico/htdocs/js/indico/jquery/declarative.js

```diff
@@ -186,4 +187,11 @@
             confirmed.then(execute);
         });
     }
+
+    function setupMathJax() {
+        var $elems = $('.js-mathjax');
+        if ($elems.length) {
```

Comment:
is that check actually needed? unless the `mathJax` method is implemented in a really awful way it should simply do nothing when called on an empty jquery object